pack/tdc/tdcc.json

```diff
@@ -13,6 +13,313 @@
+        "clues": null,
+        "code": "11507",
+        "double_sided": true,
+        "encounter_code": "one_last_job",
+        "encounter_position": 7,
+        "faction_code": "mythos",
+        "flavor": "The O'Bannions nicked Tillinghast's shipment. It's up to you to either talk Naomi into a deal, or \"convince\" her it's not worth the trouble.",
+        "illustrator": "Borja Pindado",
+        "name": "The O'Bannion Gang",
+        "pack_code": "tdcc",
+        "position": 7,
+        "quantity": 1,
+        "text": "Naomi O'Bannion gains: \"[action] Spend up to 4 resources: <b>Parley.</b> Test [intellect] or [agility] (5). This test gets -1 difficulty for each resource spent as part of this action's cost. For each point you succeed by, place 1 of your clues on Naomi O'Bannion.\"\n<b>Objective</b> - When damage is placed on Naomi O'Bannion, advance.\n<b>Objective</b> - If there are 3 clues on Naomi O'Bannion, advance.",
+        "type_code": "act"
+    },
+    {
+        "back_flavor": "As the conflict erupts into outright bloodshed, you locate a small crate labeled \"Tillinghast Endeavors\" in a back room. Keeping low to the ground, you scamper through the fight and back out to the cold evening air. The sound of shattering glass and heavy blows emanates from the building behind you.\n\"Aw, applesauce,\" says a voice above you. A compact figure clad in black slides down a rappelling line and removes their mask. \"I had a whole heist planned, you know. An informant. A calling card, even,\" Ruby says, tossing a yellow rose on the pavement. \"I didn't think you could just walk in and ask politely, but, well, I guess you won fair and square.\"",
+        "back_name": "Gang Wars",
+        "back_text": "<b>(→R1)</b> <i>(page 5)</i>.",
+        "clues": null,
+        "code": "11508",
+        "double_sided": true,
+        "encounter_code": "one_last_job",
+        "encounter_position": 8,
+        "faction_code": "mythos",
+        "flavor": "You've found yourself squarely in the center of a gang war!",
+        "illustrator": "Borja Pindado",
+        "name": "Face the Music",
+        "pack_code": "tdcc",
+        "position": 8,
+        "quantity": 1,
+        "text": "Each [[Criminal]] enemy with clues on it gains aloof.\n[action] Remove 1 clue from a [[Criminal]] enemy at your location: <b>Parley.</b> That enemy deals its damage to another enemy at your location.\n<b>Objective</b> - When the round ends, if there is at least 1 [[Elite]] enemy in the victory display, advance.",
+        "type_code": "act"
+    },
+    {
+        "back_flavor": "\"Tillinghast Esoterica - purveyor of sundry delights, exotic treasures, wondrous baubles, and innumerable secrets.\" - Recent ad in the <i>Arkham Advertiser</i>",
```

Comment:
Not sure if these should be `<em>` tags to set off as text that should be non-italicized or not; forget where we landed on that.

Reply:
Semantically, `i` is correct here